app/helpers/builds_helper.rb

```diff
@@ -21,14 +21,6 @@ def git_ref_and_sha_for(build, make_link: false)
   end
 
   def build_status_badge(build)
-    if job = build.docker_build_job
-      status_badge(job.status)
-    elsif build.docker_repo_digest
-      status_badge("succeeded")
-    elsif (Job::VALID_STATUSES - ["succeeded"]).include?(build.external_status)
-      status_badge(build.external_status)
-    else
-      "not built"
-    end
+    status_badge build.status
```

Comment:
what will no status look like ?

Reply:
build.status always returns a string, defaults to `"not built"` - same as before